components/landing/TheQuickStart/PrerequisitesForLinuxMac.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <div class="prerequisites">
+    <div class="prerequisites__section">
+      <p>Run the following command and follow the onscreen instructions:</p>
+      <pre class="code prerequisites__code">curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh</pre>
+
+      <p>
+        See <AppLink v-bind="rustupLink">
+          https://rustup.rs/
+        <!-- eslint-disable-next-line vue/singleline-html-element-content-newline -->
+        </AppLink> for more information and details on <AppLink v-bind="rustLangInstallLink">other installation methods</AppLink>.
+      </p>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import { CtaClickedEventProp } from '~/constants/segment'
+
+interface TrackedLink {
```

Comment:
We already have a [`General Link` interface](https://github.com/Qiskit/qiskit.org/blob/main/constants/appLinks.ts#L8) we can reuse instead of creating a new interface. This interface has the `label` as a mandatory value, but we can do it optional if we need it

Reply:
Good catch, thank you!

Resolved in https://github.com/Qiskit/qiskit.org/pull/2904/commits/7c13de28f1e43a06e07c016c6b588cb144fcfa3d